package.json

```diff
@@ -1,52 +1,48 @@
 {
   "name": "lytics-ember-components",
-  "version": "0.4.2",
+  "version": "0.0.0",
+  "description": "The default blueprint for ember-cli addons.",
   "directories": {
+    "doc": "doc",
     "test": "tests"
   },
   "scripts": {
-    "start": "ember server",
     "build": "ember build",
-    "test": "ember test"
-  },
-  "repository": {
-    "type": "git",
-    "url": "https://github.com/lytics/ember-components.git"
+    "start": "ember server",
+    "test": "ember try:testall"
   },
+  "repository": "",
```

Comment:
`ember init` clobbered the repository field. This will need to be reverted to 

```json
"repository": {
  "type": "git",
  "url": "https://github.com/lytics/ember-components.git"
}
```